misc/docs/source/packages/index.rst

```diff
@@ -46,6 +46,7 @@ The functionality is provided through the following packages:
    obspy.io.nied.knet
    obspy.io.pdas
    obspy.io.reftek
+   obspy.io.resp
```

Comment:
Are you also suggesting that NRL class inherit from RemoteBaseClient or HTTPClient?